ovh/design-system
# fix(pagination): Component goes out of bounds

**Bug Description:** Pagination - Goes out of bounds _[Chromium, Safari]_

**Steps to verify:** 
1) Open `OsdsPagination` component on `Chromium` or `Safari`.
2) Click on disabled `previous` arrow or `next` arrow.
3) The component should not go out of bounds anymore.
